.github/workflows/tiobe.yaml

```diff
@@ -0,0 +1,32 @@
+name: TIOBE Quality Checks
+
+on: [pull_request]
+
+jobs:
+  TICS:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+        # We could store the report from the regular run, but this is cheap to do and keeps this isolated.
+      - name: Set up Python
+        uses: actions/setup-python@v5
+      - name: Install dependencies
+        run: pip install tox~=4.2 coverage[toml]
+      - name: Generate coverage report
+        run: |
+          tox -e unit
+          coverage xml
+          # Annoyingly, the coverage.xml file needs to be in a .coverage folder.
+          rm .coverage
+          mkdir .coverage
+          mv coverage.xml .coverage/
+
+      - name: TICS GitHub Action
+        uses: tiobe/tics-github-action@v3
+        with:
+          mode: qserver
+          viewerUrl: https://canonical.tiobe.com/tiobeweb/TICS/api/cfg?name=default
+          ticsAuthToken: ${{ secrets.TICS_AUTH_TOKEN }}
```

Comment:
TICS is a clever little name that probably gets everyone confused...

then again, if we're already using this secret, so be it.

Reply:
We could change it (something in the secret here is broken - it works in my repo, but not here). Their suggestion was `TICSAUTHTOKEN` and I suggested that we break it up with the underscores. Maybe `TIOBE_AUTH_TOKEN` would be clearer? TICS is their product name, but I think we are generally calling it "TIOBE" or "TIOBE quality checks".